### PR TITLE
Update tested python version

### PR DESCRIPTION
**Pull request by matthiaskoenig**
* fix minimal tested version in documentation

### CODE REVIEW

File: INSTALL.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ For installation help, please use the `Google Group
 <http://groups.google.com/group/cobra-pie>`_. For usage instructions, please see
 the `documentation <https://cobrapy.readthedocs.org/en/latest/>`_.
 
-We only test against Python 3.6+, however, Python 3.4 or higher and even 2.7
+We only test against Python 3.7+, however, Python 3.4 or higher and even 2.7
```
Comment (cdiener) [inline INSTALL.rst]: Probably want to remove 2.7 here as this definitely does not work anymore.

**Review by cdiener — CHANGES_REQUESTED** [review]
Good catch!

**Comment by matthiaskoenig** [comment]
requested changes have been implemented.
Best M

**Review by cdiener — APPROVED** [review]
Thanks!

**Pull request merged**